examples/CExample/README.md

```diff
@@ -0,0 +1,51 @@
+## Example Description
+
+This C example demonstrates basic usage of the Web5 SDK functions through the C bindings. Specifically, it showcases:
+
+1. Resolving a `did:dht` identifier
+2. Printing the resolved DID document or an error message
+
+This example demonstrates how to resolve a DHT-based DID using the Web5 SDK in C. It uses the `did_dht_resolve` function with a hardcoded DID URI and the default gateway. The program then prints either the resolved DID document or an error message to the console.
+
+
+## Running the Example
+
+To run this C example:
+
+1. Ensure you have a C compiler installed (e.g. GCC) and [Just](https://github.com/casey/just) command runner.
+
+2. Navigate to the root directory of the project where the main Justfile is located.
+
+3. Generate the C dynamic library by running:
+
+   ```
+   just bindc
+   ```
+
+   This command will build the necessary C bindings for the Web5 SDK.
+
+4. Navigate to the `examples/CExample` directory where the example-specific Justfile is located.
```

Comment:
this step is a little non-sensical since we're already within the `examples/CExample` directory

Reply:
oh I see, but `just bindc` is at the root level of the project, never mind this makes sense